src/Benchmarks/runtimeconfig.template.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Those just process jobs - `BenchmarksServer`'s jobs clone the repo and run `Benchmarks`, and `BenchmarkClient`'s jobs invoke `wrk` to generate load. They aren't doing any critical work in terms benchmarking per se.